Pod/Classes/DateConvenience.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  DateConvenience.swift
+//  Pods
+//
+//  Created by Jason Grandelli on 11/14/16.
+//
+//
+
+import Foundation
+
+public extension Date {
+    // MARK: Units
+    public var components: DateComponents? {
+        return Calendar.current.dateComponents([.era, .year, .month, .day, .hour, .minute, .second, .weekday, .weekdayOrdinal, .quarter, .weekOfMonth, .weekOfYear, .yearForWeekOfYear, .nanosecond, .calendar, .timeZone], from: self)
+    }
+
+    public static func monthSymbols() -> [String] {
+        return DateFormatter().monthSymbols
+    }
+    
+    public static func monthSymbol(month: Int) -> String? {
+        guard month < monthSymbols().count else { return nil }
+        
+        return monthSymbols()[month]
```

Comment:
Well this is a convenience accessor for array index essentially. `DateFormatter().monthSymbols` is just an array. I'd be concerned about actually changing this where the array starts at 1 OR that we shift the incoming index by one and the end consumer not being aware we are treating this array differently. I think though that a possibly better design would be to remove this method all together and just have the `monthSymbols()` fund. The downside there is that the consumer then has to guard against out of bounds